core/CMakeLists.txt

```diff
@@ -67,6 +67,12 @@ if (AARCH64)
   include_directories(BEFORE ${PROJECT_BINARY_DIR})
 endif ()
 
+set(locvar_rel "")
+if ("${CMAKE_VERSION}" VERSION_EQUAL "3.10" OR
+    "${CMAKE_VERSION}" VERSION_GREATER "3.10")
+  set(locvar_rel "../")
```

Comment:
I don't understand why the version changes the path: please add a comment explaining

Reply:
CMP0070 is the policy. They changed the behavior to include the relative path. One could disable CMP0070, but I suppose we don't want to add more "OLD" policies, rather we're adding the "NEW" version.